heartbeat/mysql

```diff
@@ -411,9 +411,9 @@ get_read_only() {
     local read_only_state
 
     read_only_state=`$MYSQL $MYSQL_OPTIONS_REPL \
-        -e "SHOW VARIABLES" | grep read_only | awk '{print $2}'`
+        -ss -e "SHOW GLOBAL VARIABLES LIKE 'read_only'"
 
-    if [ "$read_only_state" = "ON" ]; then
+    if [ $read_only_state =~ '*ON' ]; then
```

Comment:
On Mon, Feb 22, 2016 at 06:00:05AM -0800, Kristoffer Grönlund wrote:
> >  
> > -    if [ "$read_only_state" = "ON" ]; then
> > +    if [ $read_only_state =~ '*ON' ]; then
> 
> According to http://shellcheck.net, this has two issues:
> * SC2074 Can't use =~ in [ ]. Use [[..]] instead.

I think that [[..]] is a bashism, this is a /bin/sh script.